ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -271,10 +297,16 @@ def test_replication_check_exists(self):
         Testcase to verify checks available in
         ipahealthcheck.ds.replication source
         """
+        version = tasks.get_healthcheck_version(self.master)
         result = self.master.run_command(
             ["ipa-healthcheck", "--source", "ipahealthcheck.ds.replication"]
         )
         for check in replication_checks:
+            if parse_version(version) > parse_version("0.4"):
```

Comment:
The `if parse_version(version)...` block doesn't need to be inside a for loop.

Reply:
made the changes accordingly